### ISSUE TITLE
Generated test.md file for rules should be in numeric order

### ISSUE THREAD

**Issue by VictorHom** [romejs/rome#448]
Right now, for test cases > 9, they get generated in the order of
```
example:
1.
10.
11.
12.
2.
3.
...
```
just adding a task to have it print to file in numeric order so it matches the order of the test cases

**Comment by sebmck** [comment]
In `packages/@romejs/core/test-worker/SnapshotManager.ts` we do `.sort()` on some arrays. We already have a natural sort function available as:

```
import {naturalCompare} from '@romejs/string-utils';
```

Want to submit a PR that updates the `sort()` to `.sort(naturalCompare)`?

**Comment by VictorHom** [comment]
Thanks. I'll open up a pr

**Comment by bitpshr** [comment]
Resolved by #453, this can be safely closed.
